vmdb/config/api.yml

```diff
@@ -136,6 +136,13 @@
     - :subcollection
     :methods: *70174834086080
     :klass: MiqEvent
+  :features:
+    :description: Product Features
+    :options:
+    - :subcollection
+    - :show
```

Comment:
why are we expanding features by default ?  they can be returned if asked for via expand=features.